packages/dashboard/src/components/broadcastsTable.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  DataGrid,
+  GridColDef,
+} from "@mui/x-data-grid";
+import { useRouter } from "next/router";
+import React from "react";
+
+import { useAppStore } from "../lib/appStore";
+import { monospaceCell } from "../lib/datagridCells";
+
+
+interface Row {
+  id: string;
+  name: string;
+  updatedAt: string;
+  triggeredAt: string;
+}
+
+const baseColumn: Partial<GridColDef<Row>> = {
+  flex: 1,
+  sortable: false,
+  filterable: false,
+  renderCell: monospaceCell,
+};
+
+declare module "@mui/x-data-grid" {
+  interface FooterPropsOverrides {
+    hasNextPage: boolean;
+    hasPreviousPage: boolean;
+    onNextPage: () => void;
+    onPreviousPage: () => void;
+    status: "a" | "b";
```

Comment:
Can you explain this code, why are you overriding the footer props and what is the purpose of the `status` property?

Reply:
Apologies, forgot to remove this block of code after using `usersTable.tsx` as reference. Will remove them!